__tests__/app.test.js

```diff
@@ -21,3 +28,22 @@ describe("CORE: GET /api/topics", () => {
       });
   });
 });
+
+describe("Task 10: CORE: GET /api/users", () => {
+  it("responds with array of objects of users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+
```

Comment:
need a length check. If this array is empty, the test will still pass by accident